src/HotChocolate/ElasticSearch/src/Data.ElasticSearch.Driver/Filters/Handlers/Comparable/ElasticSearchComparableGreaterThanHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using HotChocolate.Data.Filters;
+using HotChocolate.Language;
+using HotChocolate.Types;
+
+namespace HotChocolate.Data.ElasticSearch.Filters.Comparable;
+
+public class ElasticSearchComparableGreaterThanHandler : ElasticSearchComparableOperationHandler
+{
+    /// <inheritdoc />
+    public ElasticSearchComparableGreaterThanHandler(InputParser inputParser) : base(inputParser)
+    {
+    }
+
+    /// <inheritdoc />
+    protected override int Operation => DefaultFilterOperations.GreaterThan;
+
+    /// <inheritdoc />
+    public override ISearchOperation HandleOperation(ElasticSearchFilterVisitorContext context, IFilterOperationField field,
+        IValueNode value, object? parsedValue)
+    {
+        return parsedValue switch
+        {
+            double doubleVal => new RangeOperation<double>(context.GetPath(), ElasticSearchOperationKind.Filter)
```

Comment:
I added all expect decimal and ulong. As far as I got the docs, generally elastic search supports it. But I only found double properties on the NEST query classes. 

And I did not add ulong, because this way I was able to reuse long operations for all whole number value types